src/comps/ProductInfo.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Product from '../types/Product'
+import {
+    Dialog,
+    DialogTitle,
+    DialogContent,
+    DialogContentText,
+    DialogActions,
+    Button,
+    CardMedia
+} from '@mui/material';
+
+interface CartItemProps {
+    product?: Product;
+    open: boolean;
+    onOrderClick: (product: Product) => void;
+    onCloseClick: () => void;
+}
+
+const ProductInfo: React.FC<CartItemProps> = ({ product, open, onCloseClick, onOrderClick }) => {
+
+    return (
+        <Dialog
+            open={open}
+            onClose={onCloseClick}
+            dir="rtl"
+        >
+            <DialogTitle id="info-dialog-title">
+                {product?.name}
+            </DialogTitle>
+            <DialogContent>
+                <DialogContentText id="info-dialog-description">
+                    {product?.description}
+                </DialogContentText>
+                <DialogContentText>
+                    {`מחיר: ${product?.price}`}
+                </DialogContentText>
+                <CardMedia
```

Comment:
this should probably not be `CardMedia` since it is not inside a `Card` component